### DISCUSSION TITLE
Upgrade from 3.3.0 to 3.4.0 fails due to mongodb error

### DISCUSSION THREAD

**Discussion by amoughit** [litmuschaos/litmus#4476] [General]
Hello, I've launched the upgrade using litmus-cluser-scope-3.4.0.yaml file after modifying the DB_SERVER only but I'm getting the following errors in litmusportalserver pod:  

` {"error":"(IndexKeySpecsConflict) An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2,  ││ unique: true, key: { environment_id: 1 }, name: \"environment_id_1\", partialFilterExpression: { is_removed: false } }, existing index: { v: 2, unique: true, key: { environment_id: 1 }, name: \"environment_id_1\" }","file":"/gql-server/pkg ││ /database/mongodb/init.go:270","func":"github.com/litmuschaos/litmus/chaoscenter/graphql/server/pkg/database/mongodb.(*MongoClient).initAllCollection","level":"fatal","msg":"failed to create indexes for environments collection","time":"202 ││ 4-03-01T15:02:22Z"}  `

Can someone help with this issue please ?

**Comment by ispeakc0de** [comment]
Hi @abdnah ,
Please follow this thread to solve this issue https://kubernetes.slack.com/archives/CNXNB0ZTN/p1709642903703839?thread_ts=1709639195.145829&cid=CNXNB0ZTN

**Reply by amoughit** [reply]
Thanks @ispeakc0de, will try this and update here.

**Reply by amoughit** [reply]
@ispeakc0de , unfortunately I don't have account in kubernetes.slack and I'm not able to create one :(

**Reply by amoughit** [reply]
![image](https://github.com/litmuschaos/litmus/assets/65241517/a534c709-926a-4890-802b-120d2a19d9f0)

**Reply by amoughit** [reply]
@ispeakc0de, would you please help with the mongodb issue ?
